src/components/image/image.ts

```diff
@@ -275,6 +275,12 @@ export class SbbImageElement extends LitElement {
    * Set an aspect ratio
    * default is '16-9' (16/9)
    * other values: 'free', '1-1', '1-2', '2-1', '2-3', '3-2', '3-4', '4-3', '4-5', '5-4', '9-16'
+   * If the aspectRatio value is set to 'free', the component sets the CSS declaration to
```

Comment:
As we soon will release version 1.0, I think we should "sbb-"-prefix the very last not prefixed CSS var in the project :) this is causing a breaking change but it is ok for now. Furthermore, you could add a specific entry in the jsdoc section of the SbbImageElement class with "@cssprop":
```
@cssprop [--sbb-image-aspect-ratio=auto] - TBD...
```

With that it appears in readme.md and is therefore well documented and part of the public API.

see e.g. logo component for an example

Reply:
Thanks for the hint, I have indeed overlooked the documentation by @cssprop. It's added now. 

I also renamed the variable. Since it is freshly introduced, it should not cause any breaking change? Or are you referring to --image-border-radius?

Reply:
yes, I just realized it during re-review :D Probably I will create a follow-up PR for that. BTW, the image is, besides some minimal refactorings, still in the state you guys implemented it :)